src/devhub/devhub.js

```diff
@@ -142,7 +142,7 @@ async function mainSeeds() {
           <td><code>${record.command}</code></td>
           <td><time>${seedDuration}</time></td>
           <td><time>${seedFreshness} ago</time></td>
-          <td>${record.count}</td>
+          <td>${record.count.toLocaleString()}</td>
```

Comment:
FWIW, I've recently switched from toLocaleString to manual formatting for dates, because local-dependent formatting produced really weird dates (I mean, I think it picked US local)

https://github.com/tigerbeetle/tigerbeetle/pull/2672

So let's maybe remove dependency on user's locale here and pass `en-US`? 

I also personally like just spaces as the separator:

![image](https://github.com/user-attachments/assets/957c5ceb-6b69-46a1-831e-c738e221db59)

so we might also want to do `/,/&nbsp;` on the result. 

Though, LGTM as is!